migrate-the-clay-components-from-v2-to-v3.md

```diff
@@ -0,0 +1,23 @@
+# Migrating clay components from v2 to v3
+
+Noteable Changes
+
+## clay-button
```

Comment:
Yeah I agree. I think we should use this file to _over provide_ information and ideas behind changes from the devs themselves. And then later I think we need to take this information and condense it and filter it a bit to add on the official docs site.

Reply:
yeah yeah, are some of my thoughts on this, especially in the component documentation part, it will be manual work but it is necessary to build good documentation and show our thoughts. I'm excited about that 🙂